designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ScenarioActivitySpec.scala

```diff
@@ -0,0 +1,484 @@
+package pl.touk.nussknacker.ui.api
+
+import io.restassured.RestAssured.`given`
+import io.restassured.http.ContentType
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import io.restassured.response.{Response, ValidatableResponse}
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.engine.build.ScenarioBuilder
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.test.{NuRestAssureExtensions, NuRestAssureMatchers, RestAssuredVerboseLogging}
+import pl.touk.nussknacker.ui.api.helpers.TestCategories.Category1
+import pl.touk.nussknacker.ui.api.helpers.{
+  NuItTest,
+  NuTestScenarioManager,
+  TestProcessingTypes,
+  WithMockableDeploymentManager
+}
+
+import java.nio.charset.StandardCharsets
+import java.util.UUID
+
+class ScenarioActivitySpec
+    extends AnyFreeSpecLike
+    with NuItTest
+    with WithMockableDeploymentManager
+    with NuTestScenarioManager
+    with NuRestAssureExtensions
+    with NuRestAssureMatchers
+    with RestAssuredVerboseLogging {
+  import ScenarioActivitySpecAsserts._
+
+  val scenarioName      = ProcessName(UUID.randomUUID().toString)
+  val commentContent    = "test message"
+  val wrongScenarioName = ProcessName("wrongProcessName")
+  val fileContent       = "very important content"
+  val fileName          = "important_file.txt"
+
+  val scenario: CanonicalProcess = ScenarioBuilder
```

Comment:
1. let's get rid of `CanonicalProcess` (we can make this field private to satisfy Idea ;)) 
2. `scenario` -> `exampleScenario`?

Reply:
Done